### ISSUE TITLE
[ubuntu]Can't log in to company's microsoft accounts, stuck on sign in screen

### ISSUE THREAD

**Issue by czgtest** [Azure/azure-cli#10923]
Repro on : Ubuntu 18.04 LTS

follow the doc to  [Install Azure CLI with apt](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli-apt?view=azure-cli-latest ) 

after Manual install instructions,Run the Azure CLI with the az command. To sign in, use the az login command.
1. open azure page
2.Enter company email address
![image](https://user-images.githubusercontent.com/35018765/67183266-9c93d200-f413-11e9-8754-cd4cf2729959.png)

3. System hangs 

![image](https://user-images.githubusercontent.com/35018765/67183335-c5b46280-f413-11e9-9370-78e25426f949.png)

**Comment by jiasli** [comment]
This doesn't look like a CLI issue. `msft.sts.microsoft.com` may be blocked by your company's policy. Can you directly access `https://msft.sts.microsoft.com/` in the browser? 
- If not, please contact your IT department to troubleshoot it
- If so, please raise a Support Request to Azure Active Directory via Azure Portal with public internet

**Comment by qianwens** [comment]
Close this issue since there is no response for more than 1 month, please help create a new issue if you still have questions.
